ui/src/app/pages/access/data-access-requirements.tsx

```diff
@@ -423,9 +401,6 @@ export const getEligibleModules = (
     fp.filter((module: AccessModule) => isEligibleModule(module, profile)),
     fp.map(getAccessModuleConfig),
     fp.filter((moduleConfig) => moduleConfig.isEnabledInEnvironment),
-    fp.filter((moduleConfig) =>
```

Comment:
removing this on its own would cause ERA to **always** appear here instead of never.  We don't want that.  We could hard-code "if ERA, then filter out" but we'd have to remove it in the near future as we remove all other ERA references.

I see a better way we can do this: L425 uses `isEnabledInEnvironment` - this is our mechanism to remove a module from *all* UI decisions.  That is now what we want for ERA in all environments!  Following this back to how it's defined, we use the config value `enableEraCommons` from the API.  This is already `false` in Preprod, so we can trust that this won't mess things up.

So I would like to see these additional things before it's safe to remove this line:
* enableEraCommons is false in all environments
* UI Unit Tests are updated to account for that